perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -17,7 +17,17 @@
 
 # true or false to run e2e tests on installedPackage
 RUN_E2E_TESTS_ON_PACKAGE=$1
-readonly INTEGRATION_TEST_TIMEOUT=40m
+
+# Pass "true" to skip few non-essential tests.
+# By default, this script runs all the integration tests.
+SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE=$2
+
+if [ "$SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE" == true ]; then
+  GO_TEST_SHORT_FLAG="-short"
+  echo "Setting the flag to skip few un-important integration tests..."
+fi
+
+readonly INTEGRATION_TEST_TIMEOUT=60m
```

Comment:
Does this need to be increased even when SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE is true?

Reply:
Nope, currently it take around 35 minutes.

Reply:
Shall we set different values based on flag?